src/lightning_app/core/work.py

```diff
@@ -570,3 +568,10 @@ def stop(self):
         self._calls[latest_hash]["statuses"].append(stop_status)
         app = _LightningAppRef().get_current()
         self._backend.stop_work(app, self)
+
+    def _check_run_is_implemented(self) -> None:
+        if not is_overridden("run", instance=self, parent=LightningWork):
+            raise TypeError(
+                f"The work `{self.__class__.__name__}` is missing the `run()` method. This is required. Implement it"
+                " first and then call it in your Flow."
```

Comment:
Thank you so much @awaelchli for picking this up.

I really liked the message here: https://github.com/Lightning-AI/lightning/pull/14760/files#diff-7fc1f959c169d77f9190069631cd8e51de55be8cc19214c92440cee6cb42f171R129-R130, with a minor "Please", and also "work" -> "Work" (just like `Flow` is used instead of `flow`). I know this is very minor, but this is just one of those errors that is going to reach to the user a lot of times.

Probably:

```
f"The Work `{self.__class__.__name__}` is missing the `run()` method. This is required. Please implement it"
                " first and then call it in your Flow. You can refer to the docs here: https://lightning.ai/lightning-docs/basics.html?highlight=lightningwork on some basics around `LightningWork`"
```

(I'm not sure if including the docs link is a good idea here, but just pointing them to the place where they can learn how to implement the run method will help)

Nothing major, and will have to be done in another PR, but I'm just nit-picking here. (please don't mind) ❤️ 

cc: @Felonious-Spellfire on his opinion as well! I'm not a native English speaker, so I'm sorry if I'm wrong :)

Reply:
Feel free to open a PR with your ideas. Adding the doc link to the message is risky business. We haven't considered that so far as there is an open problem of how to guarantee valid links over time, so not sure if we are ready to do that just yet.